lexer.js

```diff
@@ -398,6 +398,61 @@ function tryParseObjectDefineOrKeys (keys) {
         }
         else break;
 
+        // `if (` IDENTIFIER$2 `in` EXPORTS_IDENTIFIER `&&` EXPORTS_IDENTIFIER `[` IDENTIFIER$2 `] ===` IDENTIFIER$1 `[` IDENTIFIER$2 `]) return` `;`?
+        if (ch === 105/*i*/ && source.startsWith('f ', pos + 1)) {
```

Comment:
Note: I copied `'f '` from other parts of the code, but I think it could also be `f(`?

Reply:
Nice catch, yes we should probably allow the missing space.